src/App.jsx

```diff
@@ -1,11 +1,106 @@
 import './App.css';
+import React from "react";
+import worldmap from "./assets/world_map.png";
+import axios from "axios";
+import CountryTile from "./Components/CountryTile.jsx";
+import color from "./helpers/color.js";
+import mathRound from "./helpers/mathRound.jsx";
 
 function App() {
+    const [countries, setCountries] = React.useState([]);
+    const [search, setSearch] = React.useState('');
+    const [countryData, setCountryData] = React.useState({});
```

Comment:
Als je een duidelijk discriminator wilt hebben tussen of het al wel/niet gevuld is, zou ik gaan voor een default waarde als `undefined`. Dan kun je je check verderop wat versimpelen
```suggestion
    const [countryData, setCountryData] = React.useState();
```